components/resource_metering/src/recorder/mod.rs

```diff
@@ -66,9 +69,12 @@ pub trait SubRecorder {
     /// The default period is 5 minutes.
     fn cleanup(&mut self) {}
 
+    /// This function is called before the [Recorder] thread pause.
+    fn pause(&mut self) {}
```

Comment:
I can't find where `pause` calls are.

Reply:
Updated here:

https://github.com/tikv/tikv/pull/11085/files#diff-04b1040b59943ee29d9d3784a92fe7d3ab98d3e6bbca74d98b157bea7ce27619R179-R181

Reply:
Current implementation still has one problem, wait for me to change it.

Reply:
done
I immediately updated summary_enable in `pause()` and `reset()`: [L75-L95](https://github.com/tikv/tikv/pull/11085/files#diff-5e357e4548054d86d9e08d8dfff7906b62f7ee8f8334be336ea4c783176097c9R75-R95)